ehsm_kms_service/payload_checker.js

```diff
@@ -81,7 +81,7 @@ const cryptographic_params = {
     keyid,
     ciphertext: {
       type: PARAM_DATA_TYPE.BASE64,
-      maxLength: MAX_LENGTH,
+      maxLength: (MAX_LENGTH+30)*4/3,

```

Comment:
why +30？  iv+MAC? how about to align it to 32 bytes?

Reply:
It represents: b64 len: 4 * ceil(input_length / 3)